.github/actions/detect-env/action.yml

```diff
@@ -66,17 +70,35 @@ runs:
     - name: Detect tooling
       shell: bash
       run: |
-        if ${{ inputs.newrelic || false }}; then
+        if ${{ inputs.newrelic != '' }}; then
           echo "newrelic=${{ inputs.newrelic }}" >> $GITHUB_ENV
         elif [ -f newrelic.js ]; then
           echo "newrelic=true" >> $GITHUB_ENV
         else
           echo "newrelic=false" >> $GITHUB_ENV
         fi
+    - name: Detect build path
+      shell: bash
+      run: |
+        if "${{ inputs.dist_path != '' }}"; then
+          echo "dist_path=${{ inputs.dist_path }}" >> $GITHUB_ENV
+          echo "artifact_path=${{ inputs.dist_path }}" >> $GITHUB_ENV
+        elif [ "${{ env.framework }}" == "next" && ${{ env.ssr }} == true ]; then
```

Comment:
This didn't work on Brokenships and the dist_path ended up `./dist`.
In bash, multiple conditions need two brackets: 
`[[ conditional && conditional ]]` is `[ conditional ] && [ conditional ]`

Reply:
```suggestion
        elif [[ "${{ env.framework }}" == "next" && ${{ env.ssr }} == true ]]; then
```